crates/store/Cargo.toml

```diff
@@ -19,8 +19,8 @@ anyhow                 = { workspace = true }
 deadpool               = { default-features = false, features = ["managed", "rt_tokio_1"], version = "0.12" }
 deadpool-diesel        = { features = ["sqlite"], version = "0.6" }
 deadpool-sync          = { version = "0.1" }
-diesel                 = { features = ["numeric", "sqlite"], version = "2.2" }
-diesel_migrations      = { features = ["sqlite"], version = "2.2" }
+diesel                 = { features = ["numeric", "sqlite"], version = "2.3" }
+diesel_migrations      = { features = ["sqlite"], version = "2.3" }
```

Comment:
Did we want to update these?

Reply:
The version according to `Cargo.lock` was 2.3.x anyway, and this was a leftover from an attempt to get the diesel thing to work. Can roll back if you want.